src/main/java/frc/robot/subsystems/targetvision/TargetVisionIO.kt

```diff
@@ -14,55 +13,81 @@ import java.util.*
 
 interface TargetVisionIO {
 
-    class Inputs : LoggableInputs {
-        var currentTimestamp: Double = 0.0
-        var lastUpdateTimestamp: Double = 0.0
+    class TargetVisionIOInputs : LoggableInputs {
 
-        // FIXME: this isn't being logged, our turret will be useless in replay
-        var targets: List<LimelightHelpers.LimelightTarget_Retro> = mutableListOf()
+
+        var hasTargets: Boolean = false
```

Comment:
The `hasTargets` and `targetCount` properties both just duplicate information in `targets`.